Policy "CMP0135" is not known to this version of CMake for versions before 3.24

While #935 fixed #934, it seems to have inadvertently broken older versions of CMake. With CMake 3.22 I now get the following error message:
```
CMake Error at CMakeLists.txt:11 (cmake_policy):
  Policy "CMP0135" is not known to this version of CMake.
```

CMake then terminates and there is no way to build the model. Seems like we need a way to set this policy only on newer versions of CMake.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,9 @@ cmake_policy(VERSION 3.20...3.31)
 # this requires `makeinfo` which we don't want to require.
 # TODO: Remove this when we bump the minimum CMake version to 3.24 and
 # instead add DOWNLOAD_EXTRACT_TIMESTAMP TRUE to ExternalProject_Add().
-cmake_policy(SET CMP0135 OLD)
+if(POLICY CMP0135)
+    cmake_policy(SET CMP0135 OLD)
+endif()
 
 project(sail_riscv)
 

